src/command/identity/import.ts

```diff
@@ -0,0 +1,113 @@
+import Wallet from 'ethereumjs-wallet'
+import { readFileSync, statSync } from 'fs'
+import { Argument, LeafCommand, Option } from 'furious-commander'
+import { exit } from 'process'
+import { IdentityType } from '../../service/identity/types'
+import { RootCommand } from '../root-command'
+
+export class Import extends RootCommand implements LeafCommand {
+  // CLI FIELDS
+
+  public readonly name = 'import'
+
+  public readonly description = 'Import V3 wallet as a new identity'
+
+  @Argument({ key: 'path', describe: 'Path to the V3 wallet file' })
```

Comment:
See my previous comment regarding the `required` properties.

Reply:
Agreed, added the `required` property.